phpBB/phpbb/db/migration/tool/config.php

```diff
@@ -150,6 +150,11 @@ public function reverse()
 					$arguments[0],
 				);
 			break;
+
+			case 'reverse':
+				// It's like double negative
```

Comment:
Reverse of the reverse of a call is just the call itself. That's what I mean with double negative - I will fix the comment, though.